py3status/modules/fedora_updates.py

```diff
@@ -61,15 +64,15 @@ def check_updates(self):
             self._updates = 0
             self._security_notice = False
         else:
-            if not self._security_notice and self._updates != updates:
+            if self.check_security and not self._security_notice and self._updates != updates:
                 output, error = subprocess.Popen(
                     ['dnf', 'updateinfo'],
                     stdout=subprocess.PIPE,
                     stderr=subprocess.PIPE).communicate()
                 notices = str(output)
                 self._security_notice = len(self._reg_ex_sec.findall(notices))
                 self._updates = updates
-            if self._security_notice:
+            if self.check_security and self._security_notice:
```

Comment:
We don't actually need to check `self.check_security` here as `self._security_notice` is always False, but I don't think it matters.